### DISCUSSION TITLE
Assertion failed when screen is soldered

### DISCUSSION THREAD

**Discussion by toddcblank** [ClusterM/fdskey#11] [Q&A] [answered]
Not sure if this is something you might be able to help me with, but I've made two FDS Keys.  Both of them have the same behavior.  They work prior to soldering the screen on (I can load a game and play just fine), but after soldering the screen on I get an "Error :( Assertion Failed" trying to load any file.  

If you can provide any insight into what I might've done wrong or other places to look, that'd be awesome.

Thank you!

**Comment by ClusterM** [comment, marked as answer]
Hi. Which firmware version are you using?

**Reply by toddcblank** [reply]
I thought it was the latest but I just noticed you released 1.2 last week, I'll try that quick

**Reply by toddcblank** [reply]
works like a charm now!

**Reply by ClusterM** [reply]
It was the main fix in the 1.2 :) exFAT support was broken.

**Reply by toddcblank** [reply]
It's really interesting to me that it worked without the screen soldered in though, would have never thought it was an issue with exfat support
